lib/browser.js

```diff
@@ -1,14 +1,21 @@
 var racer = require('racer');
 var BCSocket = require('browserchannel/dist/bcsocket-uncompressed').BCSocket;
 
-var CLIENT_OPTIONS = {{clientOptions}};
+const DEFAULT_CLIENT_OPTIONS = {
```

Comment:
We should be consistent about either using ES6 `let`/`const` or using ES5-compatible `var`.

I'm OK with dropping ES5 since we're major versioning, but then let's switch to `let` and `const` everywhere in this repo.